components/WarningNote/WarningNoteRecap/WarningNoteRecap.tsx

```diff
@@ -24,8 +27,34 @@ const WarningNoteRecap = ({
     return <Spinner />;
   }
 
+  const displayReviews =
+    Array.isArray(warningNote.warningNoteReviews) &&
+    warningNote.warningNoteReviews.length > 0 &&
+    formName != 'Warning Note Created';
```

Comment:
I think this may need to be `!==`, but even better this feels like something that might work better as a separate function, something like:

```ts
const displayReviews = shouldDisplayReviews(warningNote, formName); // returns `true` or `false`
```